src/components/MenuItem.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+export class MenuItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bread: "Standard",
+      isHot: false,
+      hotPrice: 0.2,
+      total: this.props.price
+    };
+  }
+
+  addBread(event) {
+    let breadName = this.props.breads[event.target.value].name;
+    let breadPrice = this.props.breads[event.target.value].price;
+    let total = this.props.price + breadPrice;
+
+    this.setState({
+      bread: breadName,
+      total: !this.state.isHot ? total : total + this.state.hotPrice
+    });
+  }
+
+  addHot() {
+    const hotPrice = this.state.hotPrice;
+    let isHot = this.state.isHot ? false : true;
+
+    this.setState({
+      isHot: isHot,
+      total: isHot ? this.state.total + hotPrice : this.state.total - hotPrice
+    });
+  }
+
+  action() {
+    this.props.addToOrder(
+      this.props.name,
+      this.state.total,
+      this.state.bread,
+      this.state.isHot
+    );
+  }
+
+  render() {
+    const { name, price, breads } = this.props;
+
+    return (
+      <div className="menu__item">
+        <p>
+          {name}
+          <span className="menu__item-price">
+            &pound;{this.state.total.toFixed(2)}
+          </span>
+        </p>
+        <select value={this.state.value} onChange={this.addBread.bind(this)}>
```

Comment:
In `App.js` you bind `this` in the constructor but in this file they are all done inline. Would be good to keep them consistent (all in the constructor) 👍

Reply:
It only needs to be bound once, usually in the constructor. Then you just call it normally in the `onChange` with `this.addBread`

Reply:
Quite right mate, this worked great - smashed them in the constructor 👍